src/utils/__tests__/formatTimestampAsUtc.test.ts

```diff
@@ -0,0 +1,12 @@
+import { formatTimestampAsUtc } from '..';
+
+describe('formatTimestampAsUtc', () => {
+  it('should not modify a UTC timestamp', () => {
+    const timestamp = '2022-07-10T14:59:03.853Z';
+    expect(formatTimestampAsUtc(timestamp)).toEqual(timestamp);
+  });
+  it('should not add Z to a non-UTC timestamp', () => {
```

Comment:
```suggestion
  it('should add Z to a non-UTC timestamp', () => {
```
Tiny nitpick, but the "should not" seems like it should just be "should" right?

Reply:
Great catch!  Fixing.